src/__mock__/countComment.js

```diff
@@ -0,0 +1,9 @@
+const comments = (commentList) => {
+  let counter;
+  commentList.forEach((comment, index) => {
+    counter = index + 1;
+  });
+  return counter;
+};
+
+module.exports = comments;
```

Comment:
- [ ] The following method counts the number of elements in an array which is different than what a counter function for this project would do. You can implement the counter function as a pure separate method which can be used in the tests directly as it's a pure function.

Reply:
okey thanks